src/Makefile

```diff
@@ -344,7 +345,8 @@ endif
 
 ifeq ($(RAYLIB_BUILD_MODE),RELEASE)
     ifeq ($(PLATFORM),PLATFORM_WEB)
-        CFLAGS += -Os
+        #CFLAGS += -Os
```

Comment:
I prefer to keep the `-Os` on web instead of `-O3`, from [emscripten documentation](https://emscripten.org/docs/optimizing/Optimizing-Code.html#how-to-optimize-code):

> Building with -O3 or -Os can produce an ever better build than -O2, and are worth considering for release builds. -O3 builds are even more optimized than -O2, but at the cost of significantly longer compilation time and potentially larger code size. -Os is similar in increasing compile times, but focuses on reducing code size while doing additional optimization.

Reply:
Same. In my project -O3 runs significantly faster than -Os and the extra build time/size is a price I am willing to pay. But this change wasn't meant to be part of the PR.